src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -180,18 +178,18 @@ public void validateIntegrationTestClassNames() throws Exception {
                         .that(matcher.matches())
                         .isTrue();
 
-                final String check = matcher.group(1);
-                assertWithMessage("Unknown check '" + check + "' in test file: " + filename)
+                final String test = matcher.group(1);
+                assertWithMessage("Unknown test '" + test + "' in test file: " + filename)
```

Comment:
```suggestion
                assertWithMessage("Unknown test '" + test + "' in corresponding file: " + filename)
```
kind of double trouble with `test` name as now its already clear from the context

Reply:
not good, sorry, should stay as is.